docs/tutorials/IndexerWriter.md

```diff
@@ -0,0 +1,112 @@
+# Conduit Indexer Writer
+
+This document explains how to configure and deploy Conduit to populate a
+PostgreSLQ DB suitable to serve as a backend for the Indexer API.
+
+Note: Windows is not currently supported by algod.
+
+The components you will configure and deploy are:
+* Algod configured as a blockchain follower.
+* PostgreSQL database.
+* Conduit configured to pull data from algod and push it to PostgreSQL
+
+```mermaid
+graph LR;
+    algod["algod"]
+    pe["postgresql Exporter"]
+    algodimp["algod Importer"]
+    db["PostgreSQL DB"]
+    indexer["Indexer REST API Service"]
+    algod-->algodimp
+    subgraph "Conduit Pipeline"
+        algodimp-->pe;
+    end
+    pe-->db;
+    db-->indexer
+```
+
+## Algod
+
+To source data from the blockchain algod is used. It will be controlled by
+Conduit in order to run synchronously. In this way it is able to store a
```

Comment:
I don't have a good alternative suggestion, but the word "synchronously" on its own doesn't mean anything to me here.